### PR TITLE
MAIN Add layout, collections, and new icons

### PR DESCRIPTION
**Pull request by herman-rogers**

### CODE REVIEW

File: src/MedicalCard/dateFormat.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const monthNames = [
```
Comment (lanwen) [inline src/MedicalCard/dateFormat.js]: should it be prepared for localization?

File: src/MedicalCard/detailedIcon.style.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    font-size: 26px;
+    color: #575756;
+  }
+
+  .icon-detail {
+    text-align: center;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 1;
+    color: #575756;
```
Comment (lanwen) [inline src/MedicalCard/detailedIcon.style.js]: i see a lot of colors inlined - should they be organized like a pallet in one place?

File: stories/simpleLoader.story.jsx [hunk 3]
```diff
@@ -1,23 +1,13 @@
...
-storiesOf("Animations", module).addWithJSX(
+storiesOf("Animations", module).add(
   "Spinner",
-  withInfo("A Spinner")(() => (
-    <div style={container}>
-      <SimpleLoader />
-      <SimpleLoader isCentered />
+  withInfo("A simple spinner for showing loading progress")(() => (
+    <div>
+      <Spinner isCentered />
```
Comment (lanwen) [inline stories/simpleLoader.story.jsx]: why not just `centered`?

File: src/MedicalCard/detailedIcon.style.js (hunk 2 above)
Comment (herman-rogers) [reply src/MedicalCard/detailedIcon.style.js]: So an inlined color would be, like 
```html
<div style={color: #23432} </div>
```

So I'm slightly confused but I'm going to interpret this as, why don't we have one location where we import the colors (like a palette, or a custom theming thing). And if that's the case then its a good point. Probably should be its own thing but Christos has also mentioned that before so might be an idea.

File: src/MedicalCard/dateFormat.js (hunk 1 above)
Comment (herman-rogers) [reply src/MedicalCard/dateFormat.js]: Cool so translations are next on my todo list after some minor style changes in the emergency service. I’ll start here for translations with another PR to make these general.

**Review by lanwen — APPROVED** [review]

**Pull request merged**